templates/repository/conf/identity/identity.xml.erb

```diff
@@ -1,20 +1,633 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
 <!--
-  ~ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
-	~
-	~ WSO2 Inc. licenses this file to you under the Apache License,
-	~ Version 2.0 (the "License"); you may not use this file except
-	~ in compliance	with the License.
-	~ You may obtain a copy of the License at
-	~
-	~ http://www.apache.org/licenses/LICENSE-2.0
-	~
-	~ Unless required by applicable law or agreed to in writing,
-	~ software distributed under the License is distributed on an
-	~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-	~ KIND, either express or implied. See the License for the
-	~ specific language governing permissions and limitations
-	~ under the License.
--->
-
-<%= scope.function_template(["wso2is/#{@product_profile}/identity.xml.erb"]) %>
\ No newline at end of file
+~ Copyright (c) 2011, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
update license.

Reply:
Not needed since only some of the configs where updated.